src/components/UploadingJobsDisplay/UploadingJobsDisplay.js

```diff
@@ -1,62 +1,88 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
 
 import {
   withStripes,
   stripesShape,
 } from '@folio/stripes/core';
+import { Layout } from '@folio/stripes/components';
 
-import {
-  UPLOADED,
-  UPLOADING,
-  FAILED,
-} from './components/FileItem/fileItemStatuses';
+import * as fileStatuses from './components/FileItem/fileItemStatuses';
+import EndOfList from '../EndOfList';
 import FileItem from './components/FileItem';
-import {
-  createFileDefinition, // eslint-disable-line
-  prepareFilesToUpload, // eslint-disable-line
-  uploadFiles,
-} from './utils/upload';
+import createUrl from '../../utils/createUrl';
+import * as API from './utils/upload';
+
+const DEFAULT_TIMEOUT_BEFORE_FILE_DELETION = 0;
+
+const NoUploadedFilesMessage = () => (
```

Comment:
This function is used once, can we move the implementation to the place of the invocation?

Reply:
I thought about it. I think it makes sense eventually, so will do this.